NineChronicles.Standalone/GraphTypes/NodeExceptionType.cs

```diff
@@ -1,4 +1,5 @@
 ﻿using GraphQL.Types;
+using Libplanet.Standalone;
```

Comment:
Is this using needed?

Reply:
It is needed because `NodeException` now in `Libplanet.Standalone`.